src/autopas/containers/linkedCells/traversals/C04SoACellHandler.h

```diff
@@ -350,17 +349,18 @@ inline void C04SoACellHandler<ParticleCell, PairwiseFunctor, DataLayout, useNewt
       }
     }
   }
+  int id = 0;
   for (unsigned long x = 0ul; x <= _overlap[0]; ++x) {
     for (unsigned long y = 0ul; y <= _overlap[1]; ++y) {
-      for (unsigned long z = 0ul; z <= _overlap[2]; ++z) {
+      for (unsigned long z = 0ul; z <= _overlap[2]; ++z, ++id) {
```

Comment:
```suggestion
      for (unsigned long z = 0ul; z <= _overlap[2]; ++z) {
```
this was just for debugging